presto-orc/src/main/java/com/facebook/presto/orc/OrcWriteValidation.java

```diff
@@ -746,28 +747,6 @@ private void build(ImmutableList.Builder<ColumnStatistics> output)
         }
     }
 
-    private static class CountStatisticsBuilder
```

Comment:
This is now an independent top level class. It will be used for the `byte` key type.